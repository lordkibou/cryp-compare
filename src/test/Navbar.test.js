import renderer from 'react-test-renderer';
import Navbar from '../scenes/navbar'
import { BrowserRouter } from 'react-router-dom';
//If the change is expected you can invoke Jest with jest -u to overwrite the existing snapshot.

it('Testing Navbar Scene', () => {
  const component = renderer.create(
    <BrowserRouter>
        <Navbar/>
    </BrowserRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});