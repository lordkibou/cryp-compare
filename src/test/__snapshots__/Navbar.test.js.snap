// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Navbar Scene 1`] = `
<header
  className="header"
>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="Logo-CrypCompar"
        className="logo"
        src="logo.png"
      />
    </a>
  </div>
</header>
`;
